components/mixins/experience-event/events/clicklink.schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/clicklink",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Click Link",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "web.webInteraction.clickLink",
```

Comment:
Should this be web.webinteraction.linkClicks (on the wiki)?

Reply:
you are correct its linkClicks . I corrected this